src/modm/math/utils/bit_operation.hpp

```diff
@@ -34,7 +34,7 @@ namespace modm
 	 * 0xab => 0xba
 	 * \endcode
 	 */
-	modm_always_inline uint8_t
+	constexpr modm_always_inline uint8_t
```

Comment:
Let's remove all the `modm_always_inline`, since constexpr already implies `inline` and inlining should be left up to the compiler.

Reply:
Done.